### ISSUE TITLE
Selenium filling in fields with masks __.__.__.___.__

### ISSUE THREAD

**Issue by KristofVanKriekingen** [sebastianbergmann/phpunit#1423]
I have an issue and can't seem to solve it or find any documentation about it.
I found an example online here:
http://www.angelwatt.com/coding/masked_input.php

It's the SSN field: ...-..-.... it works somewhat the same field as I'm working with. Whenever you fill something in with selenium it hops towards the end.
I also tried to sendkeys 1 by 1 or use tabs etc. But it just hops towards the end of the field.

Is there anything that I can do to fill in fields like this?

```
$this->webDriver->get('http://www.angelwatt.com/coding/masked_input.php');
sleep(3);          
$this->webDriver->findElement(WebDriverBy::xpath('//*[@id="ex-3"]'))->click();
$this->webDriver->getKeyboard()->sendKeys('481878483');
```

Basically this issue on SO: http://stackoverflow.com/questions/24911185/filling-in-mask-fields-with-phpunit-selenium

Thanks in advance
